File: IQCare.CCC/Entities.CCC/Tb/PatientIcfAction.cs
using Entities.Common;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entities.CCC.Tb
{
    [Serializable]
    [Table("PatientIcfAction")]
    public class PatientIcfAction : BaseEntity
    {
        [Key]
        public int Id { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int PatientId { get; set; }
        public int SputumSmear { get; set; }
        public int GeneXpert { get; set; }
        public int ChestXray { get; set; }
        public bool StartAntiTb { get; set; }
        public bool InvitationOfContacts { get; set; }
        public bool EvaluatedForIpt { get; set; }

    }
}

File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/hei/PatientIcfAction.ts
export interface PatientIcfAction {
    Id?: number;
    PatientId?: number;
    PatientMasterVisitId?: number;
    SputumSmear?: number;
    ChestXray?: number;
    StartAntiTb?: boolean;
    InvitationOfContacts?: number;
    EvaluatedForIpt?: boolean;
    DeleteFlag?: boolean;
    CreatedBy?: number;
    CreateDate?: Date;
    GeneXpert?: number;
}
